hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/leaky/LeakyCryptoTestsSuite.java

```diff
@@ -1221,14 +1221,14 @@ final HapiSpec lazyCreateViaEthereumCryptoTransfer() {
                                                         .setCallOperationType(CallOperationType.OP_CALL)
                                                         .setCallingAccount(senderAccountIdReference.get())
                                                         .setRecipientAccount(lazyAccountIdReference.get())
-                                                        .setGas(2_000_000L)
+                                                        .setGas(629_000L)
```

Comment:
Interesting...was this only passing before due to limitations in the matcher?

Reply:
Nope, the `gasLimit` of `lazyCreateTxn` got changed from `2_000_000L` to `650_000L` in another PR on `05.04.24` and it caused this assertion here to fail. 

I just fixed it cause I noticed it fails while I was working on the assertions.